app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixAvailabilityBottomSheet.kt

```diff
@@ -0,0 +1,97 @@
+import androidx.compose.material3.rememberTimePickerState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.arygm.quickfix.utils.MyAppTheme
+import com.maxkeppeker.sheets.core.models.base.rememberUseCaseState
+import com.maxkeppeler.sheets.calendar.CalendarView
+import com.maxkeppeler.sheets.calendar.models.CalendarConfig
+import com.maxkeppeler.sheets.calendar.models.CalendarSelection
+import com.maxkeppeler.sheets.calendar.models.CalendarStyle
+import java.time.LocalDate
+import java.util.Calendar
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun QuickFixAvailabilityBottomSheet(
+    showModalBottomSheet: Boolean,
+    onDismissRequest: () -> Unit,
+    onOkClick: (List<LocalDate>, Int, Int) -> Unit
+) {
+  if (showModalBottomSheet) {
+    ModalBottomSheet(
+        onDismissRequest = onDismissRequest,
+        shape = RoundedCornerShape(topStart = 16.dp, topEnd = 16.dp),
+        modifier = Modifier.testTag("availabilityBottomSheet")) {
+          Column(
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .padding(horizontal = 16.dp, vertical = 8.dp)
+                      .testTag("bottomSheetColumn"),
+              horizontalAlignment = Alignment.CenterHorizontally) {
+                val currentTime = Calendar.getInstance()
+                // State for the TimePicker
+                val timePickerState =
+                    rememberTimePickerState(
+                        initialHour = currentTime.get(Calendar.HOUR_OF_DAY),
+                        initialMinute = currentTime.get(Calendar.MINUTE),
+                        is24Hour = true // Set to false for 12-hour clock
+                        )
+                QuickFixTimePicker(timePickerState)
+                MyAppTheme {
+                  CalendarView(
+                      useCaseState =
+                          rememberUseCaseState(
+                              visible = true, onCloseRequest = { onDismissRequest() }),
+                      config =
+                          CalendarConfig(
+                              yearSelection = true,
+                              monthSelection = true,
+                              style = CalendarStyle.WEEK,
+                          ),
+                      selection =
+                          CalendarSelection.Dates { newDates ->
+                            onOkClick(newDates, timePickerState.hour, timePickerState.minute)
+                            //                            Log.d("Hi",
+                            // "$selectedHour:$selectedMinute")
+                            //                            Log.d("yo", (selectedDates).toString())
+                          })
```

Comment:
Maybe remove thoses comments on future PRs 😄

Reply:
I second that 👍🏻